src/demo/models/pond.js

```diff
@@ -69,13 +64,29 @@ const predictAllFish = state => {
   });
 };
 
+const continueText = state => {
+  return state.iterationCount === 0 ? 'Continue' : 'Complete';
```

Comment:
Should this maybe say something like "Try again!"? It is a bit weird to me that it says "Complete" then takes me back to the word picking screen.